### ISSUE TITLE
Is there any plan to add 4k display support?

### ISSUE THREAD

**Issue by WangPingGang** [plashenkov/TBX#22]
Current version is ugly in 4k display.

**Comment by plashenkov** [comment]
Yeah, I would like to fix this, but I don't know when I will be able to do this.

PRs are welcome!

**Comment by WangPingGang** [comment]
http://download.ec-software.com/delphi-developers-guide-4k.zip
here is some tips about 4k develop.
